src/cron/dailyDigest.ts

```diff
@@ -35,17 +38,20 @@ const cron: Cron = {
     await schedulePersonalizedDigestSubscriptions({
       queryBuilder: personalizedDigestQuery,
       logger,
-      handler: async ({ personalizedDigest, emailBatchId }) => {
+      handler: async ({
+        personalizedDigest: personalizedDigestWithTimezome,
+        emailBatchId,
+      }) => {
+        const { timezone = 'Etc/UTC', ...personalizedDigest } =
```

Comment:
Can we store the default timezone somewhere if it's used multiple times?

Reply:
Yes, added in 0c522af937cc06dec013abfed2363ff1fc9f16b7